addons/website_sale/views/product_views.xml

```diff
@@ -125,6 +125,28 @@
         </field>
     </record>
 
+    <!-- Product ribbon -->
+    <record id="product_ribbon_form_view" model="ir.ui.view">
```

Comment:
Side note: a new view ?

Reply:
Yes.

Reply:
Side note 2: Why ?

Reply:
To allow to edit the field 'html' (that was of type Text and now Html) as a text field.
By default, the auto generated view will use widget='html' without this one

:thinking: strange name for a field ... ;)